cms/templates/js/license-selector.underscore

```diff
@@ -0,0 +1,106 @@
+<div class="license-wrapper">
+    <label class="label setting-label" for="list-license-types">
```

Comment:
```<label>``` elements are only allowed for use with form inputs.  This references a non-widget ```div``` element.  I think a Heading element is more appropriate here.  1 more than its parent (```<h3>``` -> ```<h4>```)